pages/quiz/results.tsx

```diff
@@ -9,38 +9,20 @@ import { VerticalSpacing } from '../../src/Components/VerticalSpacing'
 import { QuizContext } from '../../src/context/QuizContext'
 import { useRouter } from 'next/router'
 import { ServicePreview } from '../index'
+import { ButtonBase } from '../../src/Components/ButtonBase'
 
 interface QuizListPageProps {
   services: Array<ServicePreview>
 }
 
-const StyledButton = styled.button`
-  align-items: center;
-  background-color: ${(props) => props.theme.colours.purple};
-  border-radius: 5rem;
-  border: 3px solid transparent;
-  color: ${(props) => props.theme.colours.white};
-  display: flex;
-  font-family: 'Catamaran', sans-serif;
-  font-weight: bold;
```

Comment:
🙌

Reply:
Not exactly related to the PR, I know, but I couldn't help myself :upside_down_face: